application/views/scripts/collections/show.php

```diff
@@ -18,7 +18,8 @@
 
             <?php if (metadata('item', 'has thumbnail')): ?>
             <div class="item-img">
-                <?php echo link_to_item(item_image('square_thumbnail', array('alt' => $itemTitle))); ?>
+                <?php $useSquareThumbnail = (option('use_square_thumbnail')) ? 'square_thumbnail' : 'thumbnail'; ?>
+                <?php echo link_to_item(item_image($useSquareThumbnail, array('alt' => $itemTitle))); ?>
```

Comment:
Since you changed item_image itself, no need to do another check here, just call item_image with a null argument